MarsOnboarding24/Pages/LanguagePage.cs

```diff
@@ -1,12 +1,150 @@
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using static System.Collections.Specialized.BitVector32;
 
 namespace MarsOnboarding24.Pages
 {
-    internal class LanguagePage
+    public class LanguagePage : CommonDriver
     {
+        IWebElement deleteButton => driver.FindElement(By.XPath("//i[@class='remove icon']"));
+        IWebElement addNewButton => driver.FindElement(By.XPath("//div[@data-tab=\"first\"]//table//thead/tr/th[3]/div[contains(text(),\"Add New\")]"));
+        IWebElement languageTextbox => driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[1]/input"));
+        IWebElement languageLevelMainDropdown => driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[2]/select"));
+        IWebElement languageLevelDropdown => driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[2]/select/option[4]"));
+        IWebElement addButton => driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[3]/input[1]"));
+
+        public void NavigateToLanguageTab(IWebDriver driver)
+        {
+            Thread.Sleep(3000);
+            Wait.WaitToBeVisible(driver, "XPath", "//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[1]/a[1]", 2);
+        }
+
+        public void ResetTest(IWebDriver driver)
+        {
+            Thread.Sleep(5000);
+
+            if (driver.FindElements(By.XPath("//form/div[2]/div/div[2]/div/table/tbody[1]")).Count != 0)
+            {
+                do
+                {
+                    deleteButton.Click();
+                    driver.Navigate().Refresh();
+                    Thread.Sleep(1000);
+                }
+                while (driver.FindElements(By.XPath("//form/div[2]/div/div[2]/div/table/tbody[1]")).Count != 0);
+            }                
+        }
+
+        public void CreateLanguageRecord(IWebDriver driver) 
+        {
+            if (driver.FindElements(By.XPath("//form/div[2]/div/div[2]/div/table/tbody[1]")).Count != 0)
+            {
+                deleteButton.Click();
+            }
+
+            addNewButton.Click();
+            languageTextbox.SendKeys("English");
+            languageLevelMainDropdown.Click();
+            Thread.Sleep(2000);
+            languageLevelDropdown.Click();
+            addButton.Click();
+        }
+
+        public void EditLanguageRecord(IWebDriver driver)
+        {
+            Thread.Sleep(3000);
+            
+            IWebElement editButton = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody/tr/td[3]/span[1]"));
```

Comment:
why is element in here? i see you are partially following the pom pattern all elements should be declared about as instance variables 


https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html